deployment/molecule/default/tests/test_all.py

```diff
@@ -18,3 +18,8 @@ def test_docker_group(host):
 def test_user(host):
     assert host.user('poadocker').exists
     assert 'docker' in host.user('poadocker').groups
+
+
+def test_logging(host):
+    assert host.file('/etc/rsyslog.d/30-docker.conf').exists
```

Comment:
Is it possible to add a check (for this file and other log files) that permissions are `0644`?

Reply:
Yes - Done.

The available assertions we can run are [here](https://testinfra.readthedocs.io/en/latest/modules.html#file).

Additionally if there are no assertions that we need, we can always use `host.run()` to run arbitrary shell command and check it's return code.